device.go

```diff
@@ -181,6 +191,19 @@ func virtioMmioBlkDeviceHandler(device pb.Device, spec *pb.Spec, s *sandbox) err
 	return updateSpecDeviceList(device, spec)
 }
 
+func virtioBlkCCWDeviceHandler(device pb.Device, spec *pb.Spec, _ *sandbox) error {
+	devPath, err := getBlkCCWDevPath(device.Id)
+	if err != nil {
+		return err
+	}
+	if devPath == "" {
```

Comment:
Nit: This code is fairly "tight" - could you spread it out with a few blank lines to make it easier to read (here and below)?